llama_index/schema.py

```diff
@@ -42,6 +42,14 @@ def class_name(cls) -> str:
         name changes.
         """
 
+    def __getstate__(self) -> Dict[str, Any]:
+        state = self.dict()
+        # Remove common unpicklable entries
+        state.pop("tokenizer", None)
+        state.pop("tokenizer_fn", None)
+        state.pop("callback_manager", None)
```

Comment:
Callback manager less-so, tokenizer yes (and especially since the tokinzer largely isn't customizable right now, but in v0.9 will be able to change the global default)

Reply:
Maybe we can keep the callback manager for now actually

Reply:
cool! yeah +1 for keeping callback manager for now until one can easily reload it

Reply:
Ok, kept the callback manager for now, it still works, but some callback managers might cause issues (we can fix that if the issue comes up on the callback manager directly)